phoneApp/keypad.js

```diff
@@ -0,0 +1,123 @@
+//Нужно визуализировать keypad.html - keypad.js
+
+//Структура виртуализации:
+// ------ Это 2 разных класса KeypadPage, ContactsPage  -----
+
+//innerHTML по максимуму
+//https://aleksandra-maslennikova.github.io/telephone-book/keypad.html
+
+//Сделайте чтобы при нажатии на кнопку цифра отобразилась
+//в <span class="numbers">
+
+//https://aleksandra-maslennikova.github.io/telephone-book/index.html
+//По клику по заголовку таблицы,
+//таблица сортировалась по соответствующему свойству
+
+class KeypadPage {
+  constructor(){
+  	this.title = 'Keypad';
+    this.buttonsValues = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '*', '0', '#', ''];
```

Comment:
please move all content specifiec part to innerHTML it's better to omit such data inside